services/meta/errors.go

```diff
@@ -55,6 +55,10 @@ var (
 	// with an existing policy.
 	ErrRetentionPolicyConflict = errors.New("retention policy conflicts with an existing policy")
 
+	// ErrRetentionPolicyDurationBelowShardDuration is returned when updating a retention
+	// policy that has a duration lower than the current shard duration.
+	ErrRetentionPolicyDurationBelowShardDuration = errors.New("retention policy duration must be greater than the shard duration")
```

Comment:
I think that's excessive... `ErrReplicationFactorTooLow` isn't prefixed with `RetentionPolicy`...

I'll let other people weigh in on the excessively long names